frontend/src/scenes/feature-flags/FeatureFlagHistory.tsx

```diff
@@ -0,0 +1,25 @@
+import { FeatureFlagTab } from 'scenes/feature-flags/FeatureFlagTabs'
+import React from 'react'
+import { FeatureFlagPageHeader } from 'scenes/feature-flags/FeatureFlag'
+import { SceneExport } from 'scenes/sceneTypes'
+import { featureFlagLogic } from 'scenes/feature-flags/featureFlagLogic'
+import { useValues } from 'kea'
+import { SceneLoading } from 'lib/utils'
+import { HistoryList } from 'lib/components/HistoryList/HistoryList'
+
+export const scene: SceneExport = {
+    component: FeatureFlagHistory,
+    logic: featureFlagLogic,
+}
```

Comment:
The way this and `featureFlagLogic` are set up now, when we move between the flag form and the history view, we reload all data always. However refactoring this to use `paramsToProps`, and scoping `featureFlagLogic` down by prop is outside of the scope of this PR I guess.

Recorded with "slow 3g" throttling:

![2022-02-15 14 40 46](https://user-images.githubusercontent.com/53387/154073628-e1c5dfb2-15e0-414a-985b-e00cb8e80f82.gif)